lib/galaxy/model/__init__.py

```diff
@@ -4176,16 +4176,17 @@ def to_dict(self, view='collection', value_mapper=None):
 class WorkflowRequestInputParameter(object, Dictifiable):
     """ Workflow-related parameters not tied to steps or inputs.
     """
-    dict_collection_visible_keys = ['id', 'name', 'value', 'type']
+    dict_collection_visible_keys = ['id', 'name', 'value', 'type', 'job_options']
     types = Bunch(
         REPLACEMENT_PARAMETERS='replacements',
         META_PARAMETERS='meta',  #
```

Comment:
Maybe I'm missing something but there shouldn't be a new column to add - you should just add a new type here for ``RESOURCE_PARAMETERS``. Or is the simple key-value setup here too simple for what you want to do?

Reply:
You are right, I completely misunderstood how this section of code should be used.